### PR TITLE
Updates docs\Bleeding_Edge.md example macro for pa tuning so that a f…

### PR DESCRIPTION
**Pull request by Jakob-Eichberger**
This pull request changes the example macro for the pa_test print found in docs\Bleeding_Edge.md to allow for very fine tuning of the pa value. It adds a pa_value and pa_range variable to the RUN_PA_TEST example macro where a pa value that should be tested around can be set. pa_range can be changed to increase/decreased the +/- range around the set pa_value. Setting pa_value to 0 deactivates this feature and prints the full pa test print starting from pa 0. 

Printed with pa_value set to zero to find the pa value that is somewhat correct:

IMG20240513215319

Printed with pa_value set to 0.025 and pa_range to 0.01. In this print, I missed the correct pa value in the picture. Currently, the default pa_range in the commit is 0.03 as that seems to be the sweet spot. 

IMG20240513215517

Issue can be found here: https://github.com/DangerKlippers/danger-klipper/issues/238

## Checklist

- [x] pr title makes sense
- [x] squashed to 1 commit
- [x] ci is happy and green

### CODE REVIEW

File: docs/Bleeding_Edge.md [hunk 1]
```diff
@@ -310,6 +330,8 @@ gcode:
 [gcode_macro RUN_PA_TEST]
 variable_bed_temp: -1
 variable_hotend_temp: -1
+variable_pa_value: 0             # Used for further tuning of pa value. If value is not 0 then the tested pa value will only be +/- (determined by the start_pa_test.range variable) around of the pavalue variable
```
Comment (YanceyA) [inline docs/Bleeding_Edge.md:333]: change "pavalue" to pa_value to reflect actual variable name.
change "start_pa_test.range" to actual variable name so it's clear to the user.

File: docs/Bleeding_Edge.md [hunk 2]
```diff
@@ -310,6 +330,8 @@ gcode:
...
+variable_pa_range: 0.03          # Only use if pa_value is set to heigher then 0. Used to set the +/- area around pa_value that should be tested
```
Comment (YanceyA) [inline docs/Bleeding_Edge.md:334]: then -> than

File: docs/Bleeding_Edge.md [hunk 3]
```diff
@@ -289,7 +289,27 @@ gcode:
     {% if flow_percent > 0 %}
     M221 S{flow_percent}
     {% endif %}
-    TUNING_TOWER COMMAND=SET_PRESSURE_ADVANCE PARAMETER=ADVANCE START=0 FACTOR=.005
+    {% if  vars.pa_value == 0 %}
```
Comment (YanceyA) [inline docs/Bleeding_Edge.md:292]: is this IF statement required? Seems redundant with the line 298 if statement?  I'm only a novice coder so appologies if it is obvious why it is required.

**Review by YanceyA — CHANGES_REQUESTED** [review]
Few minor changes to improve clarity of notes around actual variable names. Possibly a redundant If statement as noted. 

Thanks for the contribution.

**Review by YanceyA — APPROVED** [review]
No comments. Looks good to me.

**Review by rogerlz — APPROVED** [review]

**Pull request merged**